composer.json

```diff
@@ -13,6 +13,12 @@
             "email": "gregoriohc@gmail.com",
             "homepage": "https://github.com/gregoriohc",
             "role": "Developer"
+        },
+        {
+            "name": "atymic",
+            "email": "atymicq@gmail.com",
+            "homepage": "https://atymic.dev",
+            "role": "Developer"
         }
     ],
     "require": {
```

Comment:
Maybe we can drop compatibility with Laravel 5.X?

Reply:
I test to be against dropping support for no reason, considering there are 5.x versions still in support, but happy to go either way :)

Reply:
`Illuminate\Contracts\Support\DeferrableProvider` interface has been introduced at version 5.8.
Since it is used withing this PR the minimal "illuminate/" packages version should be 5.8.

Reply:
Thanks, will fix that :)